packages/govern-core/test/pipelines/helpers.ts

```diff
@@ -0,0 +1,86 @@
+import { keccak256, defaultAbiCoder, solidityPack } from 'ethers/lib/utils'
```

Comment:
Move those test-helpers to the ``govern-contract-utils`` package instead of duplicating it. (Don't forget to update the test in the ``erc3k`` package)

Reply:
... and add a unit-test test for those test-helpers cause of the complexity and importance they have.